gulpfile.js

```diff
@@ -1101,6 +1101,11 @@ function cordova_execbrowserify(file) {
     const destpath = file.replace(filename, '');
     console.log(`Include required modules in ${file}`);
     return browserify(file, { ignoreMissing: true })
+        .transform("babelify", {
+            presets: ["@babel/preset-env"],
+            sourceMaps: true,
```

Comment:
if you leave this on, it will increase built size significant. I would turn sourmaps off for production.